backend/drive/drive.go

```diff
@@ -532,6 +536,20 @@ Note that this detection is relying on error message strings which
 Google don't document so it may break in the future.
 `,
 			Advanced: true,
+		}, {
+			Name:     "service_account_folder",
+			Help:     "Use all files from the given folder as Service account file",
+			Advanced: true,
+		}, {
+			Name:     "account_disable_time",
```

Comment:
These should be prefixed with `service_account` so they sort together and the user knows they are related

`service_account_disable_time`